apps/condo/domains/acquiring/schema/RegisterMultiPaymentForInvoicesService.js

```diff
@@ -232,6 +234,19 @@ const RegisterMultiPaymentForInvoicesService = new GQLCustomSchema('RegisterMult
                     implicitFee: Big('0.0'),
                 })
 
+                const amountToPay = Big(totalAmount.amountWithoutExplicitFee)
+                    .add(Big(totalAmount.explicitServiceCharge))
+                    .add(Big(totalAmount.explicitFee))
+                if (acquiringIntegration.minimumPaymentAmount && Big(amountToPay).lt(acquiringIntegration.minimumPaymentAmount)) {
+                    throw new GQLError({
+                        ...ERRORS.PAYMENT_AMOUNT_LESS_THAN_MINIMUM,
+                        messageInterpolation: { minimumPaymentAmount: Big(acquiringIntegration.minimumPaymentAmount).toFixed(2) },
```

Comment:
While it is true that `2` is applicable for RUB, USD, EUR, and many other currencies we should support custom currencies, like BTC (why not?)

You can get radix for your current locale in acquiring integration

Reply:
may be write TODO and move it to backlog

Reply:
This is not the only place for rounding numbers to 2 digits. If we will use BTC - we will need a task to fix all places. For now, we are working only with FIAT currency